environments/unstable.env

```diff
@@ -1,3 +1,4 @@
+export BROWSER_GTM_ID=
```

Comment:
Shouldn't be needed?

Reply:
No, it has a default; but prefer being explicit.